initial-setup.spec

```diff
@@ -149,6 +149,7 @@ fi
 %{_libexecdir}/%{name}/reconfiguration-mode-enabled
 %{_unitdir}/initial-setup.service
 %{_unitdir}/initial-setup-reconfiguration.service
+%{_sysconfdir}/anaconda/conf.d/*
```

Comment:
You should also increase required version of Anaconda to avoid problem when the InitialSetup won't work thanks to the missing feature in Anaconda.

Reply:
Also not all files there are owned by InitialSetup. So please specify only the `10-initial-setup.conf` here. Otherwise you will get conflict most probably.

Reply:
Fixed.